documentation/markup.rst

```diff
@@ -618,7 +618,6 @@ an "unused" primary prompt; this is an example of what *not* to do:
 
    >>> 1 + 1
    2
-   >>>
```

Comment:
I believe that Lines 614-615 need to be updated as well. 

@ezio-melotti @terryjreedy Please verify that these lines would be better written as:
`After the last line of input or output is presented, there should be no trailing prompt. An example of correct usage is:`

Reply:
Yes, Good replacement.